website/docs/docs/cloud/dbt-cloud-ide/git-commit-signing.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Git commit signing"
+description: "Learn how to sign your Git commits when using the IDE for development."
+sidebar_label: Git commit signing
+---
+
+# Git commit signing <Lifecycle status="Enterprise" />
+
+To prevent impersonation and enhance security, you can sign your Git commits when pushing them to the repository. Using your signature, a Git provider can cryptographically verify it and mark the commit as "verified", giving people increased confidence about the origin of the commit.
```

Comment:
```suggestion
To prevent impersonation and enhance security, you can sign your Git commits before pushing them to your repository. Using your signature, a Git provider can cryptographically verify a commit and mark it as "verified", providing increased confidence about its origin.
```